src/syrupy/terminal.py

```diff
@@ -1,26 +1,42 @@
-from typing import Union
+from typing import (
+    Any,
+    Union,
+)
 
 import colored
 
+from .constants import DISABLE_COLOR_ENV_VARS
+from .utils import get_env_value
+
+
+def _is_color_disabled() -> bool:
+    return any(map(get_env_value, DISABLE_COLOR_ENV_VARS))
```

Comment:
I'm fine with this (and actually prefer it) but I thought we had a lint rule around preferring comprehension over map/reduce etc

Reply:
I don't think there's a lint rule around it, just a general case by case basis imo